modules/terrain/docs/api-reference/terrain-loader.md

```diff
@@ -73,3 +73,23 @@ The default value of `elevationDecoder` decodes a grayscale image:
   "offset": 0
 }
 ```
+
+### tesselator
+
+The choices for tesselator are as follows:
+
+`auto`:
+
+- Chooses Martini if the possible, otherwise use Delatin instead.
```

Comment:
```suggestion
- Chooses Martini if possible (if the image is a square where both height and width are powers of 2), otherwise uses Delatin instead, which has no input image limitations.
```